core/src/main/resources/org/apache/spark/ui/static/timeline-view.js

```diff
@@ -26,7 +26,10 @@ function drawApplicationTimeline(groupArray, eventObjArray, startTime) {
     editable: false,
     showCurrentTime: false,
     min: startTime,
-    zoomable: false
+    zoomable: false,
+    moment: function (date) {
+      return vis.moment(date).utcOffset(offset);
```

Comment:
Unfortunately, [moment](http://momentjs.com/docs/#/manipulating/utc-offset/) (used by `vis`) only supports the followings. Actually, this PR is using the last one, `minute` format.
```javascript
moment().utcOffset("+08:00");
moment().utcOffset(8);   // hour: -16 < x < 16
moment().utcOffset(480); // minutes
```